tests/admin_scripts/tests.py

```diff
@@ -2123,6 +2125,30 @@ def test_custom_project_template_from_tarball_by_url(self):
         self.assertTrue(os.path.isdir(testproject_dir))
         self.assertTrue(os.path.exists(os.path.join(testproject_dir, 'run.py')))
 
+    def test_custom_project_template_from_tarball_by_url_uses_Django_ua(self):
+        """
+        User-Agent is set to 'Django/version' when obtaining tarball.
+        """
+        user_agent = 'undefined'
+
+        def serve_template(request, *args, **kwargs):
+            nonlocal user_agent
+            user_agent = request.headers['User-Agent']
+            return serve(request, *args, **kwargs)
+
+        import admin_scripts.urls as test_urls
```

Comment:
Do we need an inner import? `from . import urls` should work fine.

Reply:
Ack - adding `from . import urls` as module level import.